#33431 Updated for new plugin style (#61)
postsharp/Metalama.Documentation: content/conceptual/aspects/testing/compile-time-testing.md

It's strongly recommended to target .NET 6.0 as temporary files cannot be automatically cleaned up with lower .NET versions.

> If you're referencing a project that transitively references `Metalama.Framework`, you might need to disable Metalama for the test project: `<MetalamaEnabled>false</MetalamaEnabled>`.

Reviewer: (as a new step )
```suggestion
Disable Metalama for the test project by defining the following property:

```xml
<PropertyGroup>
   <MetalamaEnabled>false</MetalamaEnabled>
</PropertyGroup>
```
